### DISCUSSION TITLE
%E7%9C%8B%E4%B9%A6/4-%E6%B8%94%E6%A8%B5%E9%97%AE%E5%AF%B9/1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#779] [Announcements]
# %E7%9C%8B%E4%B9%A6/4-%E6%B8%94%E6%A8%B5%E9%97%AE%E5%AF%B9/1

1 1 toc 摘抄理解 摘抄1 摘抄2 摘抄理解 摘抄1 大多数都不理解， 可以意会，不可言传？ 何为意会？(自己去感受然后体会？) 摘抄2 

http://localhost:8081/%E7%9C%8B%E4%B9%A6/4-%E6%B8%94%E6%A8%B5%E9%97%AE%E5%AF%B9/1.html

**Comment by pscly** [comment]
好书啊
